src/screens/Settings/ChangePassword.js

```diff
@@ -0,0 +1,127 @@
+// Setup
+import React, { Component } from "react";
+import { StyleSheet, View } from "react-native";
+
+// Vendor
+import {
+  Container, Content, H1, H2, H3, Button, Text, Item, Label, Input,
+} from 'native-base';
+
+import { Formik } from "formik";
+
+// Redadalertas
+import { colors } from "styles";
+import userServices from "services/user";
+import { checkIfLoggedIn } from "utils/user";
+
+const styles = StyleSheet.create({
+  view: {
+    flex: 1,
+    flexDirection: "row",
+    justifyContent: "center",
+    alignItems: "stretch",
+    backgroundColor: "#F5FCFF",
+    color: "#000000"
+  },
+  content: {
+    padding: 20
+  }
+});
+
+const initialValues = {
+  password: "",
+  newPassword1: "",
+  newPassword2: ""
+}
+
+export default class ChangePassword extends Component {
+  static navigationOptions = () => ({ title: "Change Password" });
+
+  state = { ...initialValues };
+
+  onSubmit = async (values) => {
+    let response;
+    try {
+      if (values.newPassword1 !== values.newPassword2) {
+        throw new Error("New passwords don't match.");
+      }
+
+      const user = await checkIfLoggedIn();
+      if (!user) throw new Error("Not logged in.");
+
+      if (true) throw new Error("Incorrect password.");
+
+      this.props.nav.navigate("SettingsPage", {
+        refresh: true
```

Comment:
Why do we need the refresh argument here?

Reply:
I believe it is currently required to ensure that upon successful password change, the updated password is passed into ChangePassword should you want to change your password again immediately.

Reply:
(It may not be necessary once we implement Redux, but right now it messes up subsequent password changes if removed.)